docs/guides/latest-updates.mdx

```diff
@@ -33,6 +33,26 @@ For more details on any of the topics covered in the blog, be sure to check [the
 
 {/* use url format /api/qiskit/release-notes/X.X to link to the specific X.X version of the release notes */}
 
+### Qiskit Runtime IBM Client 0.25.0
+
+*To see the release notes for all versions, visit the [Qiskit Runtime IBM Client release notes](/api/qiskit-ibm-runtime/release-notes).*
+
+**Highlights from the 0.25.0 (03 July 2024) version release**
```

Comment:
Here the date is `03 July 2024`, but the date for 0.24 was `2024-06-10`

Reply:
yeahhh - I put mine in the style-guidelines-approved format but .24 was in the format that the release notes used.  I'll make them consistent.